metricbeat/tests/system/test_base.py

```diff
@@ -85,28 +85,6 @@ def test_dashboards(self):
         assert exit_code == 0, 'Error output: ' + self.get_log()
         assert self.log_contains("Kibana dashboards successfully loaded.")
 
-    @unittest.skipUnless(INTEGRATION_TESTS, "integration test")
```

Comment:
the similar test was already removed in filebeats - https://github.com/elastic/beats/pull/28870, it should be removed here as well as it tests migration from 6.x to 7.x that is not supported in 8.x